modules/virt-customizing-storage-profile.adoc

```diff
@@ -6,7 +6,10 @@
 
 = Customizing the storage profile
 
-You can customize any storage profile. Use the `spec` section of the storage profile to provide specific attributes when creating an underlying persistent volume claim (PVC) for a data volume.
+You can specify default parameters using a storage profile by editing the `StorageProfile` object for the provisioner's storage class. These default parameters only apply to the persistent volume claim (PVC) if they are not configured in the `DataVolume` object.
```

Comment:
For clarity, I would suggest removing "using a storage profile" in the first sentence. I think the rest of the sentence conveys the intended meaning well enough.

Reply:
Done.